sale_packaging_price/report/sale_report.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class SaleReport(models.Model):
```

Comment:
I'm not sure, but i have seen that reports ('analysis') folders contains all files related to report.
@pedrobaeza can you help us about this??